### PR TITLE
Appsembler/extracare dogwood amir email certificate

### PR DESCRIPTION
**Pull request by amirtds**

### CODE REVIEW

**Comment by amirtds** [comment]
[(12) ExtraCare: email certificates without mailto](https://trello.com/c/BOkFlrbd/2505-12-extracare-email-certificates-without-mailto)

File: templates/certificates/_accomplishment-banner.html [hunk 1]
```diff
@@ -34,6 +37,19 @@
...
       }
+
+      var m = new mandrill.Mandrill("${settings.MANDRILL_API_KEY}");
+      var subject = decodeURIComponent("${email_share_subj}");
+      var body = decodeURIComponent("${email_share_body}");
+      var to = decodeURIComponent("${email_share_to}");
+      var params = {
+          "message": {
+              "from_email":"training.team@extracare.org.uk",
+              "to":[{"email":"amir@appsembler.com"}],
```
Comment (bryanlandia) [inline templates/certificates/_accomplishment-banner.html]: Still to:`amir@appsembler.com`

**Review by bryanlandia — CHANGES_REQUESTED** [review]
Looks good except still uses your test email address.

**Comment by amirtds** [comment]
@bryanlandia i changed to email address. (didn't want spam Extracare ppl)

**Comment by bryanlandia** [comment]
@amirtds OK, great.  To test now, you can also set up a test StudyLocation with your email address, and then assign your test user to that StudyLocation.

**Review by bryanlandia — APPROVED** [review]

**Pull request merged**